src/backend/games.ts

```diff
@@ -46,6 +47,7 @@ abstract class Game {
   abstract update(): Promise<{ status: 'done' | 'error' | 'abort' }>
   abstract isNative(): boolean
   abstract runWineCommand(args: WineCommandArgs): Promise<ExecResult>
+  abstract runner: Runner
```

Comment:
I added this for games so we can do `game.runner` instead of having to pass a `runner` string around